Overview.src.html

```diff
@@ -4178,44 +4248,96 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 method, must run these steps:
 
 <ol>
- <li><p>Let <var title>p</var> be a new promise.
+ <li><p>Let <var>p</var> be a new promise.
 
- <li><p>Let <var title>r</var> be the associated
+ <li><p>Let <var>request</var> be the associated
  <span title=concept-request>request</span> of the result of invoking the initial value of
- <code title=dom-Request>Request</code> as constructor with <var title>input</var> and
- <var title>init</var> as arguments. If this throws an exception, reject
- <var title>p</var> with it.
+ <code title=dom-Request>Request</code> as constructor with <var>input</var> and
+ <var>init</var> as arguments. If this throws an exception, reject
+ <var>p</var> with it and return <var>p</var>.
 
- <li><p>Set <var title>r</var>'s <span title=concept-request-initiator>initiator</span> to
+ <li><p>Set <var>request</var>'s <span title=concept-request-initiator>initiator</span> to
  "<code title>fetch</code>".
 
- <li><p>Set <var title>r</var>'s
- <span title=concept-request-destination>destination</span> to
+ <li><p>Set <var>request</var>'s <span title=concept-request-destination>destination</span> to
  "<code title>subresource</code>".
 
+ <li><p>Let <var>stream</var> be null.
+
  <li>
   <p>Run the following <span data-anolis-spec=html>in parallel</span>:
 
-  <p><span title=concept-fetch>Fetch</span> <var title>r</var>.
+  <p><span title=concept-fetch>Fetch</span> <var>request</var>.
 
-  <p>To <span>process response</span> for <var title>response</var>, run these substeps:
+  <p>To <span>process response</span> for <var>response</var>, run these substeps:
 
   <ol>
-   <li><p>If <var title>response</var>'s <span title=concept-response-type>type</span> is
-   "<code title>error</code>", reject <var title>p</var> with a
-   <code title>TypeError</code>.
+   <li><p>If <var>response</var>'s <span title=concept-response-type>type</span> is
+   "<code title>error</code>", reject <var>p</var> with a <code title>TypeError</code> and terminate
+   these substeps.
 
-   <li><p>Otherwise, resolve <var title>p</var> with a new <code>Response</code> object
-   associated with <var title>response</var> and a new <code>Headers</code> object whose
+   <li><p>Let <var>res</var> be a new <code>Response</code> object associated with
```

Comment:
I don't like `r` because it can be read as either one of request, response, Request and Response. I also want to use consistent names in this function - If I use `request` for a request, I wanna use `response` for a response and vice versa.

`responseObject` sounds good. Fixed.